lib/generators/decidim/templates/app.json.erb

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "<%= app_name %>",
+  "description": "TODO: Add a short description about <%= app_name %>",
+  "keywords": [
+    "small",
+    "sharp",
+    "tool"
+  ],
+  "addons": [],
+  "env": {}
```

Comment:
It's working using the defaults https://floating-coast-73437.herokuapp.com/
I'm not sure if we need to add something.